docs/open-api/indexes.yaml

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright 2024 Datastrato Pvt Ltd.
+# This software is licensed under the Apache License version 2.
+#
+
+---
+
+components:
+  schemas:
+    IndexesSpec:
+      type: array
+      items:
+        $ref: "#/components/schemas/IndexSpec"
+
+    IndexSpec:
+      type: object
+      required:
+        - indexType
+        - fieldNames
+      properties:
+        indexType:
+          type: string
+          enum:
+            - "primary_key"
+            - "unique_key"
+        name:
+          type: string
+          description: The name of the index
+        fieldNames:
+          type: array
+          description: The cols of the index
+          items:
+            $ref: "#/components/schemas/FieldNames"
+
+    FieldNames:
```

Comment:
`FieldNames` already defined in  tables.yaml:

https://github.com/datastrato/gravitino/blob/513d59156450eb24d75310784e5bf1506d321ae0/docs/open-api/tables.yaml#L293